core/pfe-core/controllers/overflow-controller.ts

```diff
@@ -44,8 +44,13 @@ export class OverflowController implements ReactiveController {
     }
   });
 
-  #ro = new ResizeObserver(() => {
-    this.#setOverflowState();
+  #ro = new ResizeObserver((entries: ResizeObserverEntry[]) => {
+    window.requestAnimationFrame((): void => {
+      if (!Array.isArray(entries) || !entries.length) {
+        return;
+      }
+      this.#setOverflowState();
+    });
```

Comment:
```suggestion
  #ro = new ResizeObserver(() => {
    requestAnimationFrame(() => {
      this.#setOverflowState();
    });
```

in my testing, we didn't need the checks. might as well save a few bytes